http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -405,6 +404,15 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
             app.customEnvironmentVariables
           )
       }
+
+      // Currently we always retry.
+      // The main failure mode here is helm install, which does not have easily interpretable error codes
+      retryConfig = RetryPredicates.standardRetryAllConfig
+      _ <- tracedRetryF(retryConfig)(
```

Comment:
I think this `retry` is in the wrong place. The `installGalaxy` / `installCustomApp` is itself retrying failures, and polling up to 20 minutes until the app becomes running. (In fact [open PR](https://github.com/DataBiosphere/leonardo/pull/2041) makes this process interruptible since we were already retrtying retries.)

Can we add the retry with `standardRetryAllConfig` _specifically_ for the `helmClient.installChart` calls? Just want to be careful about not cascading retries.

Reply:
Yeah, I noticed this morning too. Moved it